ui/App/ProfileScreen.svelte

```diff
@@ -6,50 +6,165 @@
  LICENSE file.
 -->
 <script lang="typescript">
-  import { unreachable } from "ui/src/unreachable";
+  import { onDestroy } from "svelte";
+
+  import * as router from "ui/src/router";
   import * as Session from "ui/src/session";
   import * as modal from "ui/src/modal";
-  import * as router from "ui/src/router";
+  import * as remote from "ui/src/remote";
+  import * as proxy from "ui/src/proxy";
+  import * as error from "ui/src/error";
+  import type { Project } from "ui/src/project";
+  import { isMaintainer } from "ui/src/project";
+  import * as project from "ui/src/project";
+  import * as mutexExecutor from "ui/src/mutexExecutor";
+  import * as localPeer from "ui/src/localPeer";
 
   import { Button, Icon } from "ui/DesignSystem";
 
-  import ActionBar from "ui/App/ScreenLayout/ActionBar.svelte";
   import Header from "ui/App/ScreenLayout/Header.svelte";
   import ScreenLayout from "ui/App/ScreenLayout.svelte";
-  import TabBar from "ui/App/ScreenLayout/TabBar.svelte";
+  import Error from "ui/App/ProfileScreen/Error.svelte";
+  import ProfileHeader from "ui/App/ProfileScreen/ProfileHeader.svelte";
+  import ProjectCardSquare from "ui/App/ProfileScreen/ProjectCardSquare.svelte";
+  import RequestCardSquare from "ui/App/ProfileScreen/RequestCardSquare.svelte";
```

Comment:
Could we call these `ProjectCard` and `ProjectRequestCard` respectively?

Reply:
The old `ProjectCard` is still being used by orgs. @brandonhaslegs should we make org projects use these square cards as well?

Reply:
And I inlined `RequestCardSquare` cause it was tiny

Reply:
https://github.com/radicle-dev/radicle-upstream/pull/2398/commits/485ac38bc0f582ae01cf121375c22e1245297692